Corrections, clarifications and general fixes to docs

jonaro00 opened the pull request:
I went through all documentation files in `alphatex`, `getting-started` and `tutorial-web` and fixed stuff: Corrections, clarifications, simplifications of some examples, punctuation, consistency.
Among the changes are backslashes that remove the leading newline in the alphaTex code examples, saving some vertical space.
Before: 
![image](https://user-images.githubusercontent.com/54029719/158873892-8b01d282-0b68-45e1-bc1c-a056ca1c5e30.png)
After: 
![image](https://user-images.githubusercontent.com/54029719/158873992-a37c3f57-8292-4b32-a797-23a94f76df7c.png)

Danielku15 commented on docs/alphatex/bar-metadata.mdx:
```diff
@@ -4,16 +4,16 @@ title: Bar Metadata
+Time signatures have the format `\ts Numerator Denominator`.
 
-<AlphaTexSample>{`
+<AlphaTexSample>{`\
```
Are those backslashes intentional? Feels unnecessary to me.

jonaro00 replied:
Yes, these are the ones that achieve what I showed in the pictures.

jonaro00 replied:
An alternative to backslashes everywhere would be to trim the string in the AlphaTexSample component:
`<CodeBlock>{typeof this.props.children === 'string' ? this.props.children.trim() : this.props.children}</CodeBlock>`

Danielku15 commented on docs/alphatex/tracks-staves.mdx:
```diff
@@ -4,34 +4,40 @@ title: Tracks and Staves
 1.1 2.1 3.1 4.1 |
-\\track // new track
-\\tuning A1 D2 A2 D3 G3 B3 E4 // tuning of new track
+\\track "Second" // new track
```
Leaving out the track name here was intentional to indicate that it is optional.

jonaro00 replied:
I must admit I added the "Second" more as an experiment. The reason is that I found this:
![image](https://user-images.githubusercontent.com/54029719/158893601-72823628-6385-448f-8dfd-002744ad5869.png)
to be very confusing. It looks sort of like the First track has two tunings, since there is no track name for the second tuning. It made much more sense when I saw this:
![image](https://user-images.githubusercontent.com/54029719/158893958-9288298f-b431-4845-bb42-edf0db22c38a.png)
Maybe it's just me but the (intentionally) missing name made the example look very confusing, but perhaps this is the only way to show off the optional name. _One could argue that we could also show that the short name is optional ;)_

Danielku15 commented on docs/alphatex/tracks-staves.mdx:
```diff
@@ -4,34 +4,40 @@ title: Tracks and Staves
-\\track // new track
-\\tuning A1 D2 A2 D3 G3 B3 E4 // tuning of new track
+\\track "Second" // new track
+\\tuning D2 A2 D3 G3 B3 E4 // tuning of new track
```
Also here it was intentional to have an additional string to show a difference between the two tracks on the number of strings.

jonaro00 replied:
Ah, during the confusion described in my other comment, I didn't notice that the seven stringed tuning was actually rendered in the second track. Will fix.

This just gave me an idea though. The `Tuning` part of the docs should show an example of a non-standard amount of strings tuning. Sounds good?

Danielku15 commented on docs/contributing.mdx:
```diff
@@ -78,14 +78,14 @@ The repository has a set of NPM scripts that take over actions like compiling an
+* `test` - Runs the unit test suite in Chrome
 * `test-ci` - Runs the unit test suite in Chrome Headless (used by GitHub Actions)
 * `generate-csharp` - Translates the TypeScript codebase to C#
-* `build-csharp` - compiles a C#/.net version of alphaTab using the current code state
```
The ordering in this list was intentional to be rather grouped by platform than being alphabetical or grouped by type.

jonaro00 replied:
Alright. Would also like to point out that new scripts have been added since this list was last updated. Are there any that should be included?

Danielku15 commented on docs/tutorial-web/player.mdx:
```diff
@@ -8,17 +8,17 @@ import TabItem from '@theme/TabItem';
+2. We tell alphaTab which DOM element to scroll when the cursor moves.
 3. We tell alphaTab how the cursors should look like
-4. We add some UI elements for the user to start/pause the playback and show the current position.
+4. We add some UI elements for the user to play/pause the playback and show the current position.
```
I think this should stay "start". We "start" the playback, but we do not "play the playback".

Danielku15 reviewed (changes requested):
Thanks a lot of taking the time to proof-read all these docs. Being not a native speaker and maybe sometimes lazy when it comes to docs easily degrade the quality of such docs. Having more people contributing there helps a lot. 

Some remarks regarding your changes though.

jonaro00 commented:
Reverted backslashes to instead trim strings send to `<CodeBlock>`. Added the Kotlin scripts to the list of npm scripts. Added an example of a non-standard amount of strings (5 string bass) tuning.

Danielku15 commented on docs/alphatex/tracks-staves.mdx:
```diff
@@ -4,34 +4,40 @@ title: Tracks and Staves
 \\track "Piano with Grand Staff" "pno."
     \\staff{score} \\tuning piano \\instrument acousticgrandpiano 
     c4 d4 e4 f4 |
-\
```
Have you tested that those docs pages still work? There used to be restrictions on the MDX files parser which needed backslashes here to ensure we stay in a JSX context.

jonaro00 replied:
They look and work as expected, just like the current live site but with the leading newline removed, as shown in the PR.

Danielku15 replied:
Great to hear. I remember participating in an issue around the MDX parser to improve the situation on not needing such hacks. Looks like things improved already.

Danielku15 reviewed (changes requested):
looks good mostly, just one question
